Mathlib/Algebra/Homology/ComplexShapeSigns.lean

```diff
@@ -183,6 +183,85 @@ lemma ε_up_ℤ (n : ℤ) : (ComplexShape.up ℤ).ε n = n.negOnePow := rfl
 
 end
 
+section
+
+variable (c₁ c₂)
+variable [TotalComplexShape c₁₂ c₃ c] [TotalComplexShape c₂ c₃ c₂₃] [TotalComplexShape c₁ c₂₃ c]
+
+/-- When we have three complexes shapes `c₁`, `c₂`, `c₃`, `c₁₂`, `c₂₃`, `c`, and total functors
+`HomologicalComplex₂ C c₁ c₂ ⥤ HomologicalComplex C c₁₂`,
+`HomologicalComplex₂ C c₁₂ c₃ ⥤ HomologicalComplex C c`,
+`HomologicalComplex₂ C c₂ c₃ ⥤ HomologicalComplex C c₂₃`,
+`HomologicalComplex₂ C c₁ c₂₂₃ ⥤ HomologicalComplex C c`, we get two ways to
+compute the total complex of a triple complex in `HomologicalComplex₃ C c₁ c₂ c₃`, then
+under this assumption `[Associator c₁ c₂ c₃ c₁₂ c₂₃ c]`, these two complexes
+canonical identify (without introducing signs). -/
+class Associator : Prop where
```

Comment:
The name `Associator` sounds like the name of an isomorphism, but this is a `Prop`, so maybe `Associative` is a better name?

Reply:
Thanks for the suggestion! I have changed it to `Associative`.